src/meshes/p4est_mesh.jl

```diff
@@ -1458,17 +1460,17 @@ end
 
 function update_ghost_layer!(mesh::P4estMesh)
   ghost_destroy_p4est(mesh.ghost)
-  mesh.ghost = ghost_new_p4est(mesh.p4est)
+  mesh.ghost = PointerWrapper(ghost_new_p4est(mesh.p4est))
 end
 
 
 function init_fn(p4est, which_tree, quadrant)
   # Unpack quadrant's user data ([global quad ID, controller_value])
-  ptr = Ptr{Int}(unsafe_load(quadrant.p.user_data))
+  pw = PointerWrapper(Int, unsafe_load(quadrant.p.user_data))
```

Comment:
Maybe you can add a comment about this to the respective two places in the code? I.e., why the `unsafe_load` is necessary (pointer of pointer of data) and that it is thus ok to use (since we only load/copy a scalar value)

Reply:
Thanks!